matbench/tests/test_bench.py

```diff
@@ -118,7 +118,7 @@ def test_MSONability(self):
         self.assertDictEqual(example_metadata, mb.user_metadata)
 
         with open(self.static_full_bench_json, "r") as f:
-            d_truth = json.load(f)
+            d_truth = json.loads(f.read())
```

Comment:
Why did this need to be changed?

Reply:
Also, what are the changes to the validation file? My git viewer refuses to render what the changes actually are lol

Reply:
> Why did this need to be changed?

This is so that JSON `null` is interpreted as Python `None`. The fruits of some stack exchange searching after I was having trouble getting it to be read into Python as `None`.

> Also, what are the changes to the validation file? My git viewer refuses to render what the changes actually are lol

I think the JSON file used to all be on a single line. I ran it through an auto-formatter (I think it was [this VS Code extension](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap)) and manually added `"uncertainty": null,` at 5 different locations:
![image](https://user-images.githubusercontent.com/45469701/160754340-72181bdb-e28f-48d9-a4f4-638d64459087.png)

Do you want me to try to convert this back to the single line version?

Reply:
If it is causing the errors with reading the file via json, yeah that would probably be best, esp. if VSCode did something weird with the encoding or something. 

A simpler fix to try first would be seeing if we can just

```
from monty.serialization import loadfn

d_truth = loadfn(filename)
```


Alternatively, to covert back to one line you could just try loading the json to memory using the `loads` code you have there, then `json.dump`ing the code to the same raw file

Reply:
Strange, originally I thought having the JSON on multiple lines would not have affected the issue I was seeing with `json.load(f)`, but it seems like it did. Maybe it messed with the file encoding like you mentioned.

See the commits below. I think this should address your two most recent questions fully now.